docs/GettingStarted.rst

```diff
@@ -1331,16 +1334,105 @@ manager.
+   Two entries appear: Edit and Delete.
+   
+   |image106|
+
+3. Click on ``Delete`` button --> A confirmation pop up appears.
+
+   |image107|
+
+
+4. Click ``Yes`` button in the confirmation message to accept your deletion.
+
+.. note:: As an eXo Platform user, you can only delete your own activities. If you are manager of a space, you can delete any activities related to this space.
+
+5. If you click ``Cancel`` button, nothing happens.
+
+.. _Editing-Activities-Comments:
+
+Editing activities/comments
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+
+Starting from eXo Platform 5.2, you can edit an activity you posted or a comment you wrote.
+ 
+.. _Edit_Activity:
+ 
+Editing an activity
+--------------------
+ 
+1. Click the pulldown menu on the top right of your activity : |image135|. Two entries appear: Edit and Delete.
+	
+	|image136|
+
+2. Click on ``Edit`` --> Your activity’s text appears in the editor area allowing you to make changes.
+
+    |image137|
+
+3. Make the needed changes and then click on ``Update`` button.
 
-2. Hover your cursor over the activity you want to delete, then click
-|image106| that appears.
+.. note:: The ``Update`` button remains disabled until at least one change is done.
 
-|image107|
+4. If you click the ``Cancel`` button, your changes will be ignored.
 
-3. Click **OK** in the confirmation message to accept your deletion.
+.. note:: Edition is only possible on written text. Attached images, files or link can’t be edited.
+
+              |image138|
+          
+          If the activity contains only attachments (link or files and/or images) the edit button opens the
+          editor allowing you to type a text message.
+          
+              |image139|
+              
+After saving the change you made on your activity, the activity creation timestamp will be updated by a new 
+label under your name indicating the time of the last edit:
+
+|image140|           
+          
+.. _Edit_Comment:
+ 
+Editing a comment
+------------------
+
+1. Click on the pulldown menu at the right of the comment box : |image141|. Just like for activities, two entries appear:
+   
+   |image142|
+
+2. Click on ``Edit``  --> Your comment’s text appears in the editor area allowing you to edit it.
+
+   |image143|
+   
+3. Edit your comment and then click on ``Update`` button.
+
+.. note:: The ``Update`` button remains disabled until you change the comment.
+
+4. If you click the ``Cancel`` button, your changes will be ignored.
+
+.. note:: Unlike activities, you can change an attached link/image to your comment.
+
+           |image144|
+           
+Like for activities, after saving the change you made to your comment, a text appears near your name indicating that 
+an edit has been done:
+
+|image145| 
+
+.. warning:: Activities and comments generated from other aplications such as:
+             
+				 - Adding a topic or replying to a forum discussion
```

Comment:
- closing, opening and locking a topic too will engender an automatic message. 
- Adding a poll in a topic 
- Editing an event in the agenda
- Moving a wiki page
- Editing space description
- Connexion with a new user